src/psyclone/psyGen.py

```diff
@@ -5232,14 +5260,36 @@ def lookup(self, name):
             raise KeyError("Could not find '{0}' in the Symbol Table."
                            "".format(name))
 
+    def __contains__(self, key):
+        '''Check if the given key is part of the Symbol Table.
+
+        :param str key: key to check for existance.
+        :return: Wether the Symbol Table contains the given key.
+        :rtype: bool
+        '''
+        return key in self._symbols
+
+
     @property
     def argument_list(self):
         '''
-        :return: Ordered argument list.
-        :rtype: list of Symbols
+        :return: Ordered list of arguments.
+        :rtype: list of :py:class:`psyclone.psyGen.Symbol`
         '''
         return self._argument_list
 
+    @property
+    def local_symbols(self):
+        '''
+        :return:  List of local symbols.
+        :rtype: list of :py:class:`psyclone.psyGen.Symbol`
+        '''
+        locals_list = []
+        for symbol in self._symbols.values():
+            if symbol.scope == "local":
+                locals_list.append(symbol)
+        return locals_list
```

Comment:
How about:

     return [symbol in self._symbols.values() if symbol.scope == "local"]

Reply:
👍